src/bbs2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -65,6 +68,16 @@ public class GenerateScriptCommand : CommandBase<GenerateScriptCommandArgs, Gene
         description: "The SSH port (default: 22).",
         getDefaultValue: () => 22);
 
+    public Option<string> SmbUser { get; } = new(
+        name: "--smb-user",
+        description: "The SMB user to be used for downloading the export archive off of the Bitbucket server." +
+                     $"{Environment.NewLine}" +
+                     "Note: The SMB password will also be required in order to connect to a SMB share which has to be set as an environment variable before running the script.");
+
+    public Option<string> Domain { get; } = new(
```

Comment:
What do you think of prefixing this with `Smb` to match the offer SMB-specific options?

```suggestion
    public Option<string> SmbDomain { get; } = new(
```

Reply:
addressed in 236d419ec115ff9bb55d83c20a0c22d7ebb34d94